duckduckgo/iOS
# extract find in page to its own xib and apply design update

Task/Issue URL: https://app.asana.com/0/414709148257752/1204964487427132/f
Tech Design URL:
CC:

**Description**:

Extract find in page to its own xib and apply design update.

**Steps to test this PR**:
1. Ensure find in page works as expected on phone and iPad, etc
2. Ensure design matches figma



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
